pkg/api/serialization_test.go

```diff
@@ -72,6 +72,12 @@ func fuzzInternalObject(t *testing.T, forVersion string, item runtime.Object, se
 				j.From.Kind = specs[c.Intn(len(specs))]
 			}
 		},
+		func(j *build.Build, c fuzz.Continue) {
+			c.FuzzNoCustom(j)
+			if forVersion != "v1beta1" {
+				j.ServiceAccount = ""
```

Comment:
Why is this special cased?  I would expect `ServiceAccount` to roundtrip through properly.